.github/workflows/publish_sdk.yaml

```diff
@@ -12,4 +12,27 @@ jobs:
         run: |
           chmod +x ./scripts/publish_sdk.sh
           ./scripts/publish_sdk.sh
-        shell: bash
\ No newline at end of file
+        shell: bash
+    
+      - name: Send success notification
+        uses: rtCamp/action-slack-notify@v2
+        env:
+          SLACK_CHANNEL: criteo-api-sdk-generator
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_MESSAGE: Java sdk publish succeeded. <https://github.com/criteo/criteo-api-java-sdk/actions/runs/${{github.run_id}}|Link to build>
+          SLACK_USERNAME: sdk-generation-bot
+          SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
```

Comment:
I think we should add the secret into the repo configuration

Reply:
done